src/reducers/existingThoughtMove.ts

```diff
@@ -210,9 +212,9 @@ const existingThoughtMove = (state: State, { oldPath, newPath, offset }: {
         const accumInnerChildrenOld = accumInner.contextIndex[contextEncodedOld]?.children
         const accumInnerChildrenNew = accumInner.contextIndex[contextEncodedNew]?.children
         const childrenOld = (accumInnerChildrenOld || getAllChildren(state, contextOld))
-          .filter((child: Child) => child.value !== result.value)
+          .filter((child: Child) => hashThought(child.value) !== result.hashedKey)
```

Comment:
I believe it would be more efficient to use `thoughtTransform` rather than `hashThought`, and result in the same comparison.